docs/products/opensearch/concepts/index-replication.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Replication factors in Aiven for OpenSearch®
+sidebar_label: Replication factors
+---
+
+The replication factor in Aiven for OpenSearch® determines the number of copies (replicas) of each index shard, which ensures data availability and prevents data loss.
+
+## Automatic replication factor adjustment
+
+Aiven for OpenSearch automatically adjusts replication factors in your indexes
+to maintain data availability.
+
+### How it works
+
+- **Maximum replication factor:** The maximum `number_of_replicas` is the total number
+  of nodes in your cluster minus one. This ensures that your data is replicated across
+  all available nodes.
+  - **Example:** In a 3-node cluster, the maximum `number_of_replicas` is `2`, which
+    replicates all shards across the three nodes.
+
+- **Preventing data loss:** If you set `number_of_replicas` to `0` in a multi-node
+  cluster, Aiven automatically increases it to `1`. This ensures that your data remains
+  available if one node fails.
+
+## Replication factor 0
+
+Setting the replication factor to `0` means your data has no replicas. This reduces
+storage usage but significantly increases the risk of data loss if a node in the
+cluster fails.
+
+If a node fails before the first snapshot is taken, the system cannot recover the data.
+In this case, Aiven for OpenSearch automatically recreates the missing index to restore
+operations, but the recreated index is empty. Consider this risk carefully
+before setting the replication factor to 0.
+
+In a normal node recovery scenario, Aiven for OpenSearch loads the latest snapshot to
+restore your data, minimizing data loss and speeding up recovery.
+
+:::note
+Enabling this setting comes with a reduced SLA, which must be agreed upon
+with your account manager.
+:::
+
+### Set replication factor 0
+
+Once the SLA is agreed upon, contact [Aiven support](mailto:support@aiven.io) to enable
+the setting that allows you to set the replication factor to 0.
+
+### When to use replication factor 0
+
+Consider setting the replication factor to `0` in the following scenarios:
+
+- **Non-critical environments:** Ideal for QA, testing, or development clusters where
+  potential data loss doesn’t significantly impact operations.
+- **Temporary data:** Suitable when data can be recreated or is not critical, allowing
+  you to save on storage costs.
+
+### Risks and considerations
+
+- **Data loss:** Without replicas, a node failure can result in permanent data loss. If
+  this happens, Aiven for OpenSearch automatically recreates the missing index, but
+  the recreated index is empty.
+- **Manual recovery:** If data loss occurs, data must be restored from snapshots
+  manually, which can lead to downtime.
```

Comment:
This is describing replication factor 0, so shouldn't this appear before learning how to actually enabling rep factor 0? It would appear there's some degree of repetition with the content of then _Replication factor 0_ section, and theses risks and considerations (mention of risks line 31-34, for example).

It's better to tell ppl what's in it for them before getting to tasks (which you already did to an extent). Maybe they're not interested in this and shouldn't keep on reading.

Reply:
re-ordered the section and content.